code/datums/martial/holy_crunch.dm

```diff
@@ -0,0 +1,47 @@
+/datum/martial_art/hand_of_god
+	name = "Hand of God"
+	deflection_chance = 0
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/hand_of_god/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/attack_type = pick("brute", "burn", "toxin", "knockdown")
+	if(attack_type == "brute")
```

Comment:
I would replace this whole thing with:
```
var/attack_type = list(BRUTE, BURN, TOX, STAMINA)
var/atk_verb = "power word: fist"
switch(attack_type)
   if(BRUTE)
      atk_verb = pick(blah, blah, blah)
      blah
   (same for the other types)
D.apply_damage(10, attack_type)
add_logs(A, D, " used [attack_type] attack (Hand of God) on"
```
You'd remove the switch statements and keep the logging outside of that whole shebang. This seems like a more modular way of doing things, with the exception of the slight buff that makes the knockdown do 10 stamina damage.

Reply:
switch statement yeah

Reply:
works for everything but stamina hmm